applications/product/groovyScripts/product/product/ProductServices.groovy

```diff
@@ -841,22 +867,24 @@ def checkOrderItemForProductGroupOrder() {
 /**
  * Cancle OrderItemGroupOrder
  */
-def cancleOrderItemGroupOrder() {
+Map cancleOrderItemGroupOrder() {
```

Comment:
Change to cancel would be good;) in comment also...

Reply:
I appreciate this seems a simple change and is an overall improvement, but my concern is that all these little 'fixes' will add up to something that is different to the original aim of translating the existing code to satisfy the Codenarc rules and therefore adds to the cognitive load for someone to review the changes.

My preference would be to leave existing behaviour as it is, and create some other 'good housekeeping' tickets to be applied after this PR is merged.

Reply:
That could be indeed be done later, notably because it also entails other changes in code